app/assets/javascripts/components/static/about_page.es6.jsx

```diff
@@ -2,69 +2,85 @@ class AboutPage extends React.Component {
 
   constructor(props) {
     super(props);
-    this.state = {
-      people: [
-        'daniel', 'richard', 'rocky',
-        'rachel', 'sahil', 'lancy'
-      ],
-    }
-  }
+    this.individuals = [
```

Comment:
I tried moving the data into a separate constants file, but I ran into trouble because the bio_constants.es6.jsx file is not recognizing another constants file, image_constants.es6.jsx. Is there a way to resolve this?